### PR TITLE
[PLAY-1262] Playbook Icons Styles

### PR DESCRIPTION
**Pull request by markdoeswork**
**What does this PR do?** A clear and concise description with your runway ticket url.

Runway https://runway.powerhrg.com/backlog_items/PLAY-1262

Some of our icon's props did animations, size, and other fantastic things. But the css to power these props was comming from the font awsome library. This story copied that functionality independant of font awsome 

check it out here 
https://pr3491.playbook.beta.px.powerapp.cloud/beta/icons

https://pr3491.playbook.beta.px.powerapp.cloud/samples/icons

Because some of the icons in nitro are 

- selected by class name, or are 
- adding these classnames to apply certain styles to icons 

I applies some of FA's css with original classnames to this pr

https://github.com/powerhome/playbook/pull/3491/commits/33cd559dc5de45c23d9c0ce9bd5cc82eb293e3a2#diff-48e5586ea05b8f0356ffc79e2fa9c4881f49eef0b7417843b6d80f0820db49dbR146-R232

### CODE REVIEW

File: playbook/app/pb_kits/playbook/pb_collapsible/__snapshots__/collapsible.test.js.snap [hunk 1]
```diff
@@ -28,7 +28,7 @@ exports[`html structure is correct 1`] = `
               style="vertical-align: middle; color: rgb(193, 205, 214);"
             >
               <i
-                class="pb_icon_kit far fa-fw fa-lg fa-chevron-down"
+                class="pb_icon_kit far far fa-lg fa-fw fa-fw fa-lg fa-chevron-down"
```
Comment (jasperfurniss) [inline playbook/app/pb_kits/playbook/pb_collapsible/__snapshots__/collapsible.test.js.snap:31]: @markdoeswork Why are the classes duplicated on these tests?

Comment (markdoeswork) [reply playbook/app/pb_kits/playbook/pb_collapsible/__snapshots__/collapsible.test.js.snap:31]: you know i never really looked into it, Ill go through that again now

File: playbook/app/pb_kits/playbook/pb_collapsible/__snapshots__/collapsible.test.js.snap [hunk 2]
```diff
@@ -28,7 +28,7 @@ exports[`html structure is correct 1`] = `
               style="vertical-align: middle; color: rgb(193, 205, 214);"
             >
               <i
-                class="pb_icon_kit far fa-fw fa-lg fa-chevron-down"
+                class="pb_icon_kit far fa-lg fa-fw fa-fw fa-lg fa-chevron-down"
```
Comment (jasperfurniss) [inline playbook/app/pb_kits/playbook/pb_collapsible/__snapshots__/collapsible.test.js.snap:31]: Still seeing `fa-fw fa-fw` ...  that one is twice as well.

**Comment by nidaqg** [comment]
Looks like the icon flip for 'both' is not working, as compared to prod (both react and rails)

### review env for this PR:
Screenshot 2024-07-05 at 11 07 52 AM

### prod:
Screenshot 2024-07-05 at 11 08 56 AM

**Comment by markdoeswork** [comment]
@nidaqg it works good now https://pr3491.playbook.beta.px.powerapp.cloud/kits/icon/react#icon-rotate

**Review by nidaqg — APPROVED** [review]

**Pull request merged**